### PR TITLE
Add 2021 Annual Survey Launch post

### PR DESCRIPTION
**Pull request by rylev**
The tentative plan is to go live on the 22nd of November. This post is largely a copy/paste of previous years' announcement posts, just with some relevant details changed, added, or omitted. 

I'm marking this as a draft since we're still waiting on publishing the surveys and so we don't yet have links. 

cc @badboy (for community team eyes), @nrc (as my survey co-author), @Mark-Simulacrum (for Foundation awareness since they want to market this as well)

### CODE REVIEW

File: posts/2021-11-22-survey-launch.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Launching the 2021 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the sixth year of Rust"
+---
+
+It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Community Team is pleased to announce our [2021 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future.
```
Comment (nrc) [inline posts/2021-11-22-survey-launch.md:8]:
```suggestion
It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Community Team is pleased to announce our [2021 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and establish development priorities for the future.
```

**Review by nrc — APPROVED** [review]

**Review by badboy — DISMISSED** [review]
As this is the same text we use in the past this looks good to me.
Excited for getting this launched!
Thanks for all the work!

**Comment by rylev** [comment]
This is ready to review. The tentative time for release is at 16:00 UTC on the 23rd of November, but this will be re-evaluated before shipping. If anyone wants to take the surveys and check for last minute bugs - that would be appreciated it! ❤️

**Review by nrc — DISMISSED** [review]

**Comment by nrc** [comment]
ping @badboy for re-review <3

**Review by badboy — DISMISSED** [review]
Approving the text on behalf of community.
Is there already a new planned date for this?

**Comment by rylev** [comment]
@badboy I just pushed a change. The new tentative date is today (the 6th of December).

**Review by badboy — DISMISSED** [review]

File: posts/2021-12-06-survey-launch.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+[survey]: https://surveyhero.com/c/jzesmzph
+[English]: https://surveyhero.com/c/dauv4jcd
+[Portuguese]: https://surveyhero.com/c/bhdqz3hj
+[Simplified Chinese]: https://surveyhero.com/c/7rnfv4cf
+[French]: https://surveyhero.com/c/y7hb34nm
+[Korean]: https://surveyhero.com/c/m3bi3nad
+[Spanish]: https://surveyhero.com/c/he7yvafu
+[Russian]: https://surveyhero.com/c/xix7rf7d
+[Traditional Chinese]: https://surveyhero.com/c/h7guatdt
+[German]: https://surveyhero.com/c/pq43azn4
```
Comment (badboy) [inline posts/2021-12-06-survey-launch.md:45]: image

For the German version `Rust Programmiersprache` is not properly linked, instead the url is visible after it.

Comment (rylev) [reply posts/2021-12-06-survey-launch.md]: I'll fix that - for anyone else looking here who has feedback on the survey's themselves, you can report them [here](https://rust-lang.zulipchat.com/#narrow/stream/294169-t-community.2Frust-survey-2021).

**Review by Mark-Simulacrum — APPROVED** [review]

**Review by Mark-Simulacrum — APPROVED** [review]

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**